internal/cmp/constants.go

```diff
@@ -24,5 +24,19 @@ const (
 	tier1GatewayType             = "NSX-T Tier-1 Gateway"
 	routerFirewallExternalPolicy = "GatewayPolicy"
 
+	// load balancer consts
+	ApplicationProfile       = "application-profile"
+	ServiceTypeLBHttpProfile = "LBHttpProfile"
+	tier1                    = "/infra/tier-1s/26cdb82e-0057-4461-ad4d-cddd61d77b1f"
+	networkServerID          = 1
+	size                     = "SMALL"
```

Comment:
Same question here, are these case sensitive? If yes, we need to make that it gets reflected in the documentation.

Reply:
yes its case sensitive

Reply:
Same here

Reply:
validation already added and we will get the error  "msg": "Field level validation errors: {value Small of property size is not one of the allowed values [SMALL, MEDIUM, LARGE, XLARGE, DLB]}"